raw-data/EA/EA_inventories.tsv

```diff
@@ -8112,36 +8112,36 @@ InventoryID	LanguageCode	LanguageName	Segment
 2433	kjh	Khakas (Standard)	uː
 2433	kjh	Khakas (Standard)	ɔ
 2433	kjh	Khakas (Standard)	ɔː
-2434	0	Eastern Khanty (Vakh)	p
-2434	0	Eastern Khanty (Vakh)	v
-2434	0	Eastern Khanty (Vakh)	m
-2434	0	Eastern Khanty (Vakh)	t
-2434	0	Eastern Khanty (Vakh)	s
-2434	0	Eastern Khanty (Vakh)	n
-2434	0	Eastern Khanty (Vakh)	l
-2434	0	Eastern Khanty (Vakh)	ʈʂ
-2434	0	Eastern Khanty (Vakh)	ɳ
-2434	0	Eastern Khanty (Vakh)	ɭ
-2434	0	Eastern Khanty (Vakh)	r
-2434	0	Eastern Khanty (Vakh)	c
-2434	0	Eastern Khanty (Vakh)	j
-2434	0	Eastern Khanty (Vakh)	ɲ
-2434	0	Eastern Khanty (Vakh)	ʎ
-2434	0	Eastern Khanty (Vakh)	q
-2434	0	Eastern Khanty (Vakh)	ʁ
-2434	0	Eastern Khanty (Vakh)	ŋ
-2434	0	Eastern Khanty (Vakh)	iˑ
-2434	0	Eastern Khanty (Vakh)	eˑ
-2434	0	Eastern Khanty (Vakh)	æˑ
-2434	0	Eastern Khanty (Vakh)	yˑ
-2434	0	Eastern Khanty (Vakh)	øˑ
-2434	0	Eastern Khanty (Vakh)	ɯˑ
-2434	0	Eastern Khanty (Vakh)	ɑˑ
-2434	0	Eastern Khanty (Vakh)	oˑ
-2434	0	Eastern Khanty (Vakh)	ɪ
-2434	0	Eastern Khanty (Vakh)	ʏ
-2434	0	Eastern Khanty (Vakh)	ɯ̽
-2434	0	Eastern Khanty (Vakh)	ʊ
+2434	NA	Eastern Khanty (Vakh)	p
```

Comment:
We said NA was better than "0" or "-" which is in the raw data. They are still NA because I haven't had the time to go through and search on each and try to make an informed decision of what they could be. In general though, the Glottolog codes have a higher resolution for languages (including fine-grained dialectal information), so we may find more matches there. This brings up another issue we should discuss. It seems to me that maintaining language codes in the raw data and in the InventoryID-ISO-Glottolog is unnecessary additional work. I would suggest we keep the raw data as is (so in this case we get ISO codes from the EA compilers and leave it in the raw data), but then we implement our own language code decisions by maintaining the InventoryID-ISO-Glottolog mappings, which we add to the aggregation script. There we can add our inferences for Khanty, etc., and maintain one ISO and one Glottolog code per Inventory ID. Then it is essential that the raw data contains persistent Inventory IDs.